mysql.go

```diff
@@ -137,6 +139,14 @@ func columnMapper(column *rel.Column) (string, int, int) {
 	}
 }
 
+func dropKeyMapper(typ rel.KeyType) string {
+	if typ == rel.ForeignKey {
+		return "FOREIGN KEY"
+	}
+
+	panic(fmt.Sprintf("drop key: unsupported key type `%s`", typ))
```

Comment:
can you add a test to cover this line too?

Reply:
added